miasm2/os_dep/common.py

```diff
@@ -73,14 +73,13 @@ def vm_alloc(self, vm, size, perm=PAGE_READ | PAGE_WRITE):
         @perm: permission flags (PAGE_READ, PAGE_WRITE, PAGE_EXEC or any `|`
             combination of them); default is PAGE_READ|PAGE_WRITE
         """
-        addr = self.next_addr(size+8)
-        vm.add_memory_page(addr, perm, "\x00" * (size+8),
+        addr = self.next_addr(size)
+        vm.add_memory_page(addr, perm, "\x00" * (size),
                            "Heap alloc by %s" % get_caller_name(2))
-        vm.set_mem(addr, pck64(size))
-        return addr+8
+        return addr
 
     def get_size(self, vm, ptr):
-        return upck64(vm.get_mem(ptr-8, 8))
+        return vm.get_all_memory()[ptr]["size"]
```

Comment:
As you add a new public function, can you add a docstring describing it?
I know this is not the case for every function in this module, but we (at least try to) add docstring on new implemented function. Thus, Miasm readibility gets better on average :smiley: 

For this function, there is a particularity: `ptr` must be the base address of the mapped function. If you want to have a more permissive function, you can do something like (untested but should work):
```Python
assert vm.is_mapped(ptr, 1)
data = vm.get_all_memory()
ptr_page = data.get(ptr, None)
if ptr_page is None:
    for address, page_info in data.iteritems():
        if address <= ptr < address + page_info["size"]:
            ptr_page = page_info
            break
    else:
        raise RuntimeError("Must never happen (unmapped but mark as mapped by API)")
return ptr_page["size"]
```
Note for a future PR: this function should probably reside in VmMngr implementation